packages/appeals-service-api/src/mappers/back-office.mapper.js

```diff
@@ -0,0 +1,77 @@
+const BackOfficeAppealSubmissionAggregate = require("../models/aggregates/back-office-appeal-submission.aggregate");
+const BackOfficeSubmissionEntity = require("../models/entities/back-office-submission-entity");
+
+class BackOfficeMapper {
+
+    /**
+     * 
+     * @param {AppealContactsValueObject} appealContactDetails 
+     * @param {string} appealId 
+     * @param {string[]} documentIds 
+     * @returns 
+     */
+    appealToAppealToBeSubmittedJson(appealContactDetails, appealId, documentIds) {
+        return {
+			organisations: this.#getOrganisationsJson(appealContactDetails),
+			contacts: this.#getContactsJson(appealContactDetails) ,
+			appeal: { horizon_id: "", id: appealId, failures: [] },
+			documents: this.#getDocumentsJson(documentIds)
+		}
+    }
+
+    fromJsonToBackOfficeAppealSubmission(json){
+        const organisations = json
+            .organisations
+            .map(organisationJson => new BackOfficeSubmissionEntity(
+                organisationJson.type,
+                organisationJson.horizon_id 
+            ));
+
+        const contacts = json
+            .contacts
+            .map(contactJson => new BackOfficeSubmissionEntity(
+                contactJson.type,
+                contactJson.horizon_id 
+            ));
+
+        const appeal = new BackOfficeSubmissionEntity(json.appeal.id, json.appeal.horizon_id);
```

Comment:
Just a minor but usually is more clean to pass something like "json.appeal" through the constructor instead of a granular constructor. If a refactoring makes use of one more property we don't need to refactor this call

Reply:
I hear you, and I agree. However, ideally we'd pass an appeal model here and invoke something like `getId()` in order to not expose the JSON structure but, pulling the appeal model together will be a big job, so I've run with this for now in the hopes that it can be tidied later. Is that cool?